src/Video/FrameBuffer.cpp

```diff
@@ -6,9 +6,7 @@
 
 using namespace Video;
 
-FrameBuffer::FrameBuffer(const std::vector<ReadWriteTexture*>& textures) {
-    this->textures = textures;
-
+FrameBuffer::FrameBuffer(const std::vector<ReadWriteTexture*>& textures) : textures(textures) {
     // Frame buffer object.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'textures' is assigned in constructor body. Consider performing initialization in initialization list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuseInitializationList)